git-buildkite

```diff
@@ -176,14 +184,20 @@ if [ -z "$BUILDKITE_ORGANIZATION" -o -z "$BUILDKITE_PIPELINE" ]; then
 fi
 
 # Parse flags before parsing arguments
-let i=1
-while [[ $i -le $# ]]; do
-  case "${!i}" in
+while [[ $# -gt 0 ]]; do
+  case "$1" in
     "--browse")
       browse=yes
-      shift $i
+      shift
+      ;;
+    "--force")
+      force=yes
+      shift
+      ;;
+    *)
+      break
+      ;;
   esac
-  let i=i+1
 done
```

Comment:
Just as a note, the previous code didn't quite work here due to the usage of `shift $i`. The newer version is something I see fairly often in Bash, and serves to strip out any switches, then positional args (which should be branch/commit) start at `$1` for the parts of the script below 201